pkg/operation/botanist/resource_manager.go

```diff
@@ -81,7 +83,8 @@ func (b *Botanist) DefaultResourceManager() (resourcemanager.Interface, error) {
 				corev1.ResourceMemory: resource.MustParse("30Mi"),
 			},
 		},
-		SchedulingProfile: v1beta1helper.ShootSchedulingProfile(b.Shoot.GetInfo()),
+		SchedulingProfile:            v1beta1helper.ShootSchedulingProfile(b.Shoot.GetInfo()),
+		DefaultSeccompProfileEnabled: gardenletfeatures.FeatureGate.Enabled(features.DefaultSeccompProfile),
```

Comment:
Hm, in this way the webhook gets enabled for the GRM running in the Shoot control plane.
We only need to enable it for the central GRM, right? For the Shoots we agreed to explicitly specify the seccomp profile in the Pods specs and to do not introduce the webhook.

```suggestion
```